tests/datasets/test_custom_dataset.py

```diff
@@ -29,13 +28,15 @@ def test_custom_dataset(step_lr, optimizer, tokenizer, get_model, train, mocker)
     args, kwargs = train.call_args
     train_dataloader = args[1]
     eval_dataloader = args[2]
+    tokenizer = args[3]
```

Comment:
Please add steps for running this test. What are the expected values that user has to provide for the tokenizer and other params

Reply:
The test is self-contained and will run when a user calls the test suite with python -m tests. The args here just represent the argument that were given to the train function (which is actually mocked out) by the main fine-tuning routine which we're testing (besides the custom dataset preprocessing itself). We use the arguments given to mocked train function to check them against the expected values.